draft-ietf-rats-architecture.md

```diff
@@ -91,22 +91,16 @@ a remote peer (the "Relying Party") to decide whether to consider that
 Attester a trustworthy peer or not.
 RATS are facilitated by an additional vital party, the Verifier.
 
-The Verifier appraises Evidence via Appraisal Policies and creates
-the Attestation Results to support Relying Parties in their decision
-process.
-
-This documents defines a flexible architecture with corresponding roles
-and their interaction via conceptual messages.
+This documents defines a flexible architecture consisting of attestation roles
+and their interactions via conceptual messages.
 Additionally, this document defines a universal set of terms that can be mapped to various existing and emerging Remote Attestation Procedures.
 Common topological models and the data flows associated with them, such as
-the "Passport Model" and the "Background-Check Model" are illustrated. The
-purpose is to enable readers of this document to map their current and
-emerging solutions to the architecture provided and the corresponding
-terminology defined.
-
-A common terminology that provides a well-understood semantic meaning to
-the concepts, roles, and models in this document is vital to create
-semantic interoperability between solutions and across different platforms.
+the "Passport Model" and the "Background-Check Model" are illustrated.
+The purpose is to enable readers to map their solution architecture to the
+canonical attestation architecture provided here and to define useful terminology for attestation.
```

Comment:
could be misread as "The purpose is to enable readers to ... define useful terminology for attestation." which is not the intent.

```suggestion
The purpose is to define useful terminology for attestation and enable readers to map
their solution architecture to the canonical attestation architecture provided here.
```

Reply:
If we are at it, I'd say "remote attestation" instead of "attestation". That nit aside, I think this reads better.